operations/Restore_HSM.md

```diff
@@ -2,9 +2,31 @@
 
 ## Procedure
 
-1. Place `tar` file from [Backup](Backup_HMS.md) on the system to restore.
+1. If `tar` file was placed in `s3` from [Backup](Backup_HMS.md) copy to the system to restore.

```

Comment:
If applicable, this could be added: (`ncn-mw#`),
or maybe it should to be added to all the sub items (i.e. "List objects in `s3`")